core/store/src/db.rs

```diff
@@ -415,9 +448,9 @@ impl RocksDB {
     pub fn new<P: AsRef<std::path::Path>>(path: P) -> Result<Self, DBError> {
         let options = rocksdb_options();
         let cf_names: Vec<_> = (0..NUM_COLS).map(|col| format!("col{}", col)).collect();
```

Comment:
You can use `strum::EnumIter` to implement `.iter()` for DbCol and then iterate over it instead of relying implicitly on `0..NUM_COLS`: https://github.com/Peternator7/strum/wiki/Derive-EnumIter

Reply:
Hmm this will break compatibility for sure, but now we don't need to think if adding a merge operator breaks it.

Reply:
How would it break it?

The values are fixed numbers, so when you iterate over there enum variants and do `variant as usize` you will get the same number as it is not, but strictly typed up until the point of casting

Reply:
makes sense now